packages/api/endpoints/elasticsearch.js

```diff
@@ -1,17 +1,16 @@
 'use strict';
 
 const router = require('express-promise-router')();
+const { v4: uuidv4 } = require('uuid');
 
 const log = require('@cumulus/common/log');
-const asyncOperations = require('@cumulus/async-operations');
 const { IndexExistsError } = require('@cumulus/errors');
 const { defaultIndexAlias, Search } = require('@cumulus/es-client/search');
 const { createIndex } = require('@cumulus/es-client/indexer');
 
 const { asyncOperationEndpointErrorHandler } = require('../app/middleware');
-
-const models = require('../models');
 const { getFunctionNameFromRequestContext } = require('../lib/request');
+const startAsyncOperation = require('../lib/startAsyncOperation');
```

Comment:
Can we generally import this like const { invokeStartAsyncOperationLambda } = require('../lib/startAsyncOperation') and avoid all of the extra text/typing of startAsyncOperation.invokeStartAsyncOperationLambda?